doc/source/whatsnew/v0.21.0.txt

```diff
@@ -566,7 +566,9 @@ Categorical
 - Bug in the categorical constructor with empty values and categories causing
   the ``.categories`` to be an empty ``Float64Index`` rather than an empty
   ``Index`` with object dtype (:issue:`17248`)
-
+- Bug in categorical operations on Series like ``Series.cat.set_categories``
```

Comment:
should instead refer to all ``Series.cat`` operations not preserving name

Reply:
Should I mention all the functions along with set_categories here?

Reply:
I would just refer to http://pandas.pydata.org/pandas-docs/stable/categorical.html#working-with-categories

Reply:
@jreback , the tests have passed but Travis CI build has failed. Y is that? :(